### DISCUSSION TITLE
Regarding the problem of frame 0 being predicted

### DISCUSSION THREAD

**Discussion by wyclearnpy** [talmolab/sleap#1768] [Help!]
When I use the unet network to train the data, I can predict new data. However, when I use the resnet network to train, the prediction result is 0 frames for the same data. Has anyone encountered this problem? By the way, I'm doing single-animal testing

**Comment by eberrigan** [comment]
How did the metrics for both models turn out? Thanks!

Elizabeth

**Reply by eberrigan** [reply]
@wyclearnpy please send us your configuration file so we can see the hyperparameters you use. Could you copy and paste the output in your terminal during training and inference. Thanks!

You can copy and paste your information in a reply or upload your config and a text file with your terminal output [here](https://docs.google.com/forms/d/e/1FAIpQLSfWBwcuZH8lxGyFabMAODLjSUE4YrSCYbrrnYvXCpgBqZpcJA/viewform).
